raft/fsm.cc

```diff
@@ -360,9 +360,7 @@ fsm_output fsm::get_output() {
     if (_observed._last_conf_idx != _log.last_conf_idx() ||
```

Comment:
It would be nice to limit the length of commit comment rows with 80 cols. I use 66 actually, this makes commit comments easier to read in git log.

Reply:
ok, will fix